src/core_plugins/kibana/public/visualize/wizard/_wizard.scss

```diff
@@ -0,0 +1,48 @@
+@include euiBreakpoint('xl') {
+  .visWizard {
+    display: flex;
+  }
+
+  .visWizard__column {
+    display: flex;
+    flex-direction: column;
+    padding: 0px $euiSizeXS;
+  }
+
+  .visWizard__row {
+    flex: 1;
+    background-color: $euiColorLightestShade;
+  }
+
+  .visWizard__column--small {
+    flex: 2;
+  }
+
+  .visWizard__column--large {
+    flex: 3;
+  }
+}
+
+/**
+ * 1. Hack in some padding for these panels.
+ */
+.visWizard__list--paginated--selectable,
```

Comment:
would this just be `--paginatedSelectable`?

Reply:
I did it like this because they wording makes them seem like separate modifiers. It's 'paginate' and it's 'selectable'.